src/lib/index.js

```diff
@@ -1319,6 +1347,10 @@ lib.bigFont = function(size) {
     return Math.round(1.2 * size);
 };
 
+lib.beginCap = function(str) {
+    return str.charAt(0).toUpperCase() + str.slice(1);
```

Comment:
https://github.com/plotly/plotly.js/blob/bdb69fc18f3b7787c93fff7ee1db2a634a62ee78/src/lib/index.js#L723-L725

Reply:
Done in https://github.com/plotly/plotly.js/pull/5125/commits/1efc41eb012ef653486414a616aa05de3b84f879